core/workspace_svg.js

```diff
@@ -1800,7 +1800,14 @@ Blockly.WorkspaceSvg.prototype.onMouseWheel_ = function(e) {
   }
 
   var scrollDelta = Blockly.utils.getScrollDeltaPixels(e);
-  if (canWheelZoom && (e.ctrlKey || !canWheelMove)) {
+
+  // Zoom should also be enabled by the command key on Mac devices.
```

Comment:
```suggestion
  // Zoom should also be enabled by the command key on Mac devices,
  // but not super on Unix.
```
Could you modify this comment to clarify why we need the `userAgent` check? =)